code-challenges.js

```diff
@@ -58,12 +96,40 @@ const givenValue1 = 42
 const givenValue2 = 10
 // Expected output: 8
 
+// console.log(numberOfConnections.lastIndexOf(givenValue1));
+// console.log(numberOfConnections.lastIndexOf(givenValue2));
+
+
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
 // Pseudo code:
+//create function to hold logic
+    // iterate over given array
+        //on each loop, iterate through array again for swapping
+            // if value at current index < next index
+                // swap values
+                // temp = currentIndxValue
+                // currentIndx = nextValue
+                // next index = temp
+        // return the array
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
 // Expected output: [82, 80, 79, 77, 76, 73, 72]
 
 const sanDiegoWinterTemperatures = [59, 68, 62, 59, 66, 67, 66]
 // Expected output: [68, 67, 66, 66, 62, 59, 59]
+
+// const sortTemp = (array) => {
+//     for (let i = 0; i < array.length; i++){
+//         for (let j = 0; j < array.length; j++){
+//             if (array[j] < array[j + 1]){
+//                 let temp = array[j];
+//                 array[j] = array[j + 1];
+//                 array[j + 1] = temp;
+//             }
+//         }
+//     }
```

Comment:
Nice job figuring out this logic, though I would avoid using nested for loops as they lead to increased time complexity, which means that the execution time of the code can grow exponentially with the size of the data. For a refactor, you could look into the built-in method `.sort`.